manifests/install.pp

```diff
@@ -12,6 +12,13 @@
   }
 
   if $enable_pki_core {
+    package { 'maven:3.8':
+      ensure      => installed,
+      enable_only => true,
```

Comment:
Perhaps we should force it to switch modules too. I think that can be done this way:
```suggestion
    package { 'maven dnf module':
      ensure      => '3.8',
      name        => 'maven',
```

It may also be good to add a specific acceptance test that first enabled maven 3.5 or 3.6 and then runs the module. See if it can deal with that.

And I'd also make the maven module version that it ensures a parameter.

Reply:
That was the trick, thanks!